packages/wells/README.md

```diff
@@ -164,15 +164,55 @@ const retrievedCrs = wells.items.map(well => well.wellhead?.crs)
 
 ### _Filter wells by npt events_
 
+This allows you to filter wells on associated npt events.
+
+**duration**: how long did the event last.
```

Comment:
Duration in *seconds*.

Reply:
duration is in hours. I'll specify that

Reply:
In the source data this is already stored in hours